internal/server/server_test.go

```diff
@@ -197,6 +197,9 @@ func TestSignupEnabled(t *testing.T) {
 }
 
 func TestServer_Run(t *testing.T) {
+	if testing.Short() {
```

Comment:
Out of curiosity, how slow are these?

Reply:
Seems to be anywhere from 2s to 3.5s. All the time is spent in the HTTPS sub test, and this got a lot slower recently. I suspect the time is generating private keys for the CA.  We may be able to speed up this test by generating the private key and certs ahead of time,  including them in git under `./testdata/`, and having the server use those.